data/deep missions.txt

```diff
@@ -1165,27 +1174,27 @@ mission "Deep: Project Hawking: Prime"
-			`	The scientists speak with the spaceport authorities, who begin loading several tons of seperate allows into your cargo from a platform above your ship. You sit with Isaac as you wait for them to finish.`
+			`As you approach the massive <planet> spaceport building, Pierre directs you to park on a landing pad on the third story close to the warehouse section. "There are some alloys used in the creation of ships that we need samples of for testing," Pierre explains.`
+			`	The scientists speak with the spaceport authorities, who begin loading several tons of various alloys into your cargo from an overhead platform. You sit with Pierre as you wait for them to finish.`
 			choice
-				`	"What are these alloys going to be used for?"`
-				`	"Why do we need to come all the way to Prime to get these alloys?"`
+				`	"That's a lot of different alloys. What are they going to be used for?"`
+				`	"Why did we need to come all the way to Prime to get these alloys?"`
 					goto alloys
 			
-			`	Isaac cocks his head to the side as he looks at you. "We're going to destroy them. Short and simple. We need to see how different ship alloys hold up to the technology we're working on.`
+			`	Pierre cocks his head to the side as he looks at you. "We're going to destroy them. Short and simple. We need to see how different ship alloys hold up to the technology we're working on.`
 			
 			label alloys
-			`	"We could have gotten alloys from a shipyard local to the Deep, but Betelgeuse Shipyards gets regular shipments of alloys more commonly used in ships that are sold in the south, so it's the cloest place to get everything we need."`
+			`	"We could have gotten alloys from a shipyard local to the Deep, but Betelgeuse Shipyards gets regular shipments of alloys more commonly used by shipbuilders in the Dirt Belt, so it's really the closest place to get everything we need."`
 			choice
 				`	"Do you enjoy the work you do for the Deep?"`
 				`	"How long have you been working for the Deep?"`
 			
 			label story
-			`	"Let me answer that question in a short story of my life." Isaac clears his throat and buffs up his chest a little as if preparing himself to tell a great epic. He may enjoy talking about himself a little too much...`
-			`	"I was born on Windblain in the Regor system. It's a quiet little world of little interest, but you should give it a vist some time. My family wasn't the most well off when I was a child, so I needed to work hard in my studies, which paid off immensely. Thanks to my exceptional grades in university, I was offered a government job right out of graduation. I've been working directly for the Deep for the past 35 years and counting, and I've loved every minute of it. My younger sister though... well, let's just say she wasn't so focused on being successful in school and ended up on the wrong side of the law."`
-			`	"How about you, kid?"`
```

Comment:
From the quality checklist:
`Conversations should not imply that the player is a certain gender or use gendered pronouns for the player (call them "kid" or "Captain" or <first> instead).`
~~Isaac~~ Pierre is in his 60s at this point, so I think it's fair for him to call the player a 
"kid." :)

Reply:
We have no idea how long it took to save up (in a textile mill, i think... a long time), or how quickly the player did this string.
how about a branch on the current year? if it's more than 3100 (lol), he says old-timer, and if its < 3015, he says kid?

Reply:
I'd assume that the player ends up getting their license in their late teens (19) or early-to-mid twenties; it would certainly parallel plenty of adventure stories where the main hero is around that age range, going out into a new, mysterious world with little wisdom in hand given their age.